src/AlertsManager.cpp

```diff
@@ -687,10 +703,20 @@ void AlertsManager::notifyAlert(AlertEntity alert_entity, const char *alert_enti
   json_object_object_add(notification, "icon_emoji",
 			 json_object_new_string(getAlertLevel(alert_severity)));
 
+
   if(ntop->getRedis()->get((char*)ALERTS_MANAGER_SENDER_USERNAME,
-			   alert_sender_name, sizeof(alert_sender_name)) >= 0)
+			   alert_sender_name, sizeof(alert_sender_name)) >= 0) {
+
+    strncpy(notification_username, alert_sender_name, 64);
+
+    if (alert_severity == alert_level_error) strncat(notification_username, " - NOTIFY OF ERROR", 32);
```

Comment:
avoid hard-coding strings here. Also see if you can re-use this: https://github.com/SauroPollastrini/ntopng/blob/dev/src/AlertsManager.cpp#L649